site/content/docs/4.3/components/navbar.md

```diff
@@ -66,9 +66,13 @@ Here's an example of all the sub-components included in a responsive light-theme
           <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
         </li>
       </ul>
-      <form class="form-inline my-2 my-lg-0">
-        <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+      <form class="row g-2 mt-3 mt-lg-0 pl-lg-2">
```

Comment:
This example is busted in the docs.

<img width="820" alt="Screen Shot 2020-01-09 at 1 08 39 PM" src="https://user-images.githubusercontent.com/98681/72104890-3b2c2280-32e1-11ea-87cf-afdfbd60b9f4.png">